### PR TITLE
Implementation of a restaurant website.

### PR DESCRIPTION
**Pull request by luisvinicius09**
# This simple implementation is about a restaurant pages. 👷  

### Includes:
- Homecont
- Menu content
- Contact content

> Build entirely with JS ❤️ .

### CODE REVIEW

**Review by Srjnnnn — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

Hello, @luisvinicius09. Your page fuınctionality is good. :+1:. You've used the ES6 syntax, and thanks for that. While I was reviewing you've made some changes and after I realized it I've pulled them, however, you've forgotten webpack-dev-server. Right now, I'm using the VScode live server extension to run your app on the local server. However, you need to arrange a start script for your page, which runs on the webpack-dev-server. Kindly, implement this.

Apart from this issue, I think everything is good and as required. :100: 
If you have any questions about the review, feel free to slack me. :100: 
_Eyüp Sercan UYGUR_

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "name": "jsRestaurant",
+  "version": "1.0.0",
+  "description": "This is the homepage for a restaurant.",
+  "main": "index.js",
+  "scripts": {
+    "build": "npx webpack",
+    "watch": "npx webpack --watch",
+    "server": "npx webpack serve",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (teekaytech) [inline package.json]: Kindly update the necessary run common commands like build, development in package.json inside `scripts`. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/gitignore_requirement_pathwright.png) from Pathwright)
```JSON
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @luisvinicius09 :wave:

__I must commend your efforts so far in this project. Thank you for implementing the required changes as highlighted by the last reviewer.

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] Some linter checks were not successful. Kindly ensure that all checks are running/passing successfully before you request another review.
- [ ] Please check my inline comment below and fix it as appropriate.


* * *
**Please make the required changes and submit another review request when you are done.**

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by christianotieno — APPROVED** [review]
## Project Approved 🍾 
Hello @luisvinicius09  👋🏾
Thank you for working on the required changes.👍🏾 
Congratulations. The project is now approved. 👏🏾 
All the very best in your next tasks 🥇 
### Happy coding 💻 
#### Christian Otieno (CR)

**Pull request merged**